src/font.ts

```diff
@@ -199,15 +187,15 @@ export class Font {
     let size: string;
     const sz = fontInfo.size;
     if (sz === undefined) {
-      size = Font.SIZE + 'pt ';
+      size = '';
     } else if (typeof sz === 'number') {
       size = sz + 'pt ';
     } else {
       // size is already a string.
       size = sz.trim() + ' ';
     }
 
-    const family: string = fontInfo.family ?? Font.SANS_SERIF;
+    const family: string = fontInfo.family ?? '';
```

Comment:
Same here. We want to have this method always return a valid CSS font declaration. The default family should be something like 'Arial, sans-serif' instead of just an empty string.

Reply:
done